src/app/redux/actions/youTube.actions.ts

```diff
@@ -0,0 +1,10 @@
+import { createAction, props } from '@ngrx/store';
+import { WholeDataCustom } from 'src/app/shared/types';
+ 
+export const LoadItems = createAction('[Main Page] Load Items');
+export const LoadItemsSuccess = createAction('[Main Page] Load Items Success', props<{ items: WholeDataCustom[] }>());
+export const AddToFavorites = createAction('[Main Page] Add To Favorites', props<{item: WholeDataCustom }>());
+export const DeleteFromFavorites = createAction('[Favorites Page] Delete From Favorites', props<{ id: string }>());
+export const GetDetails = createAction('[Main Page] Get Details');
+export const GetDetailsSuccess = createAction('[Main Page] Get Details Success');
```

Comment:
Why it doesn't have payload?
if we have received something, then we must transfer it for processing in the store